var/spack/repos/builtin/packages/vep-cache/package.py

```diff
@@ -0,0 +1,222 @@
+    # that one may want to avoid that
+    vep_versions = ["110", "111", "112"]
+    vep_assembly_sources = ["ensembl", "refseq", "merged"]
+    # This is an incomplete list
+    vep_species = [
+        ("bos_taurus", ["UMD3.1"]),
+        ("danio_rerio", ["GRCz11"]),
+        ("homo_sapiens", ["GRCh37", "GRCh38"]),
+        ("mus_musculus", ["GRCm38"]),
+        ("rattus_norvegicus", ["Rnor_6.0"]),
+    ]
+    # We only need to (and only can) specify an assembly for humans
+    vep_assembly_choices = [
+        assembly
+        for species, assemblies in vep_species
+        for assembly in assemblies
+        if species == "homo_sapiens"
+    ]
+
+    variant("use_vep_installer", default=True, description="Use VEP installer script to download")
+    variant("env", default=True, description="Setup VEP environment variables for this cache")
+
+    # Cache configuration options
+    variant("fasta", default=True, description="Add FASTA files to the cache")
+    variant("indexed", default=True, description="Use indexed cache")
+
+    variant(
+        "assembly_source",
+        values=vep_assembly_sources,
+        default="ensembl",
+        description="What reference genome source",
+    )
+    variant(
+        "species",
+        values=[species for species, _ in vep_species],
+        default="homo_sapiens",
+        description="Which species to download the cache for (only one at a time)",
+    )
+    variant(
+        "assembly",
+        values=[assembly.lower() for assembly in vep_assembly_choices],
+        default="grch38",
+        when="species=homo_sapiens",
+        description="Which assembly of genome to use (only needed for homo sapiens)",
+    )
+
+    # Add all species for each VEP version
+    for major, source, indexed, (species, assemblies) in itertools.product(
+        vep_versions,  # All VEP versions
+        vep_assembly_sources,  # The three VEP assembly sources
+        [True, False],  # Indexed or not
+        vep_species,  # All species with caches defined
+    ):
+        # A possibility of more than one assembly, even though most only have one
+        for assembly in assemblies:
+            version(major)
+            _vep_cache_resource(
+                version=major, species=species, source=source, assembly=assembly, indexed=indexed
+            )
+
+    depends_on("vep", type="build", when="~use_vep_installer")
+
+    @property
+    def vep(self):
+        return self.spec["vep"].package
+
+    def vep_cache_config(self, base):
+        spec = self.spec
+        satisfies = spec.satisfies
+        variants = spec.variants
+        cache_version = spec.version.up_to(1)
+        vep_version = self.vep.version.up_to(1)
+        user_root = join_path(base, "share", "vep")
+        root = user_root  # Should this be VEP install dir?
+
+        if cache_version == "default":
+            cache_version = vep_version
+
+        indexed = satisfies("+indexed")
+        cache_type = variants["source"].value
```

Comment:
I think this must be assembly_source
```suggestion
        cache_type = variants["assembly_source"].value
```